tools/taskcluster/schedule-nightly-graph.py

```diff
@@ -68,17 +78,26 @@ def as_slugid(name):
     return (task_id, task)
 
 
-if __name__ == "__main__":
+def schedule(is_staging):
     queue = taskcluster.Queue({ 'baseUrl': 'http://taskcluster/queue/v1' })
 
-    branch, head_rev = calculate_branch_and_head_rev(ROOT)
-
+    html_url, branch, head_rev = calculate_git_references(ROOT)
     params = {
-        'repository_url': 'https://github.com/mozilla-mobile/focus-android',
+        'is_staging': is_staging,
```

Comment:
Nit: I don't see this parameter used in `make_decision_task()`. Let's remove it.